config/config.go

```diff
@@ -128,9 +130,16 @@ func LoadConfiguration(c *cli.Context) error {
 	viper.AddConfigPath(filepath.Join(home, rootConfigPathName, contextsPathName))
 	viper.SetConfigName(context)
 
+	if viper.GetString("cli-version") != "" {
+		if viper.GetString("cli-version") != Version {
+			fmt.Printf("Welcome to Fn CLI %s - please check the release notes at:\n%s \n\n", Version, "https://github.com/fnproject/cli/blob/master/README.md")
+		}
+	}
+	WriteCurrentContextToConfigFile(CurrentCliVersion, Version)
```

Comment:
This doesn't seem to actually write anything - my config.yaml is unchanged.

Also, I'd change the name of "WriteCurrentContextToConfigFile" if the function is writing more than just the current context - something like "WriteConfigValueToConfigFile" maybe?